### DISCUSSION TITLE
I made a small display to show most recent detections

### DISCUSSION THREAD

**Discussion by chvvkumar** [blakeblackshear/frigate#2307] [Show and tell]
![image](https://user-images.githubusercontent.com/16548147/142899066-3e79a1fc-5fa5-46fc-a938-dd3839a73451.png)

Raspberry Pi3:

Chromium browser auto starts up with the following configuration (note the 'kiosk' option at the end of hte URL, it's needed when using the Kiosk integration mentioned below):

Autostart config file:

`/etc/xdg/lxsession/LXDE-pi/autostart
`
```
@lxpanel --profile LXDE-pi
@pcmanfm --desktop --profile LXDE-pi
@xscreensaver -no-splash
@unclutter -idle 10
@xset s off
@xset -dpms
@xset s noblank
@chromium-browser --kiosk http://hassdebian.lan:8123/small-screen/0?kiosk
HyperPixel 4.0 Square (https://shop.pimoroni.com/products/hyperpixel-4-square?variant=30138251477075)
```

Frigate:

Frigate HASS Integration: 
https://github.com/blakeblackshear/frigate-hass-integration

Frigate HASS Card: 
https://github.com/dermotduffy/frigate-hass-card

Kiosk Mode: 
https://github.com/maykar/kiosk-mode

A new lovelace dashboard for just this screen with one view in single panel mode.

I am working on using the HTTP API to get a static image and show that instead of a Lovelace dashboard but this was a quick and dirty project :)

**Comment by Minglarn** [comment]
Nice one.. Was thinking to do the same... But I ended up with a similar on my dashboard..
![image](https://user-images.githubusercontent.com/58695742/143688866-13138a55-fa6d-40e3-8afc-68c1f7d044dc.png)

**Reply by chvvkumar** [reply]
> Nice one.. Was thinking to do the same... But I ended up with a similar on my dashboard.. ![image](https://user-images.githubusercontent.com/58695742/143688866-13138a55-fa6d-40e3-8afc-68c1f7d044dc.png)

Nice! I have a smaller display so I am trying to figure out how to show the most recent snapshots from all my cameras as a slide show. I have since changed the LED to a 5" one on a RPi:

![image](https://user-images.githubusercontent.com/16548147/143718527-04b49b68-4ad6-4612-8b26-37d16417b552.png)

**Reply by Minglarn** [reply]
You could save snapshots in a special folder and let RPI show the images inside the folder...
That's really easy in HA. Just listen to the MQTT and save the snapshot. The let the RPI cycle trough the images in the folder.
